tests/garage/envs/test_repeat_action_env.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+import gym
+import numpy as np
+
+from garage.envs.wrappers import RepeatAction
+from garage.misc.overrides import overrides
+
+
+class TestRepeatAction(unittest.TestCase):
+    @overrides
+    def setUp(self):
+        self.env_repeat = RepeatAction(
+            gym.make("Breakout-v0"), n_frame_to_repeat=4)
+        self.env = gym.make("Breakout-v0")
```

Comment:
Use mocks to avoid interacting with gym

https://semaphoreci.com/community/tutorials/getting-started-with-mocking-in-python